src/site/content/en/blog/sign-in-form-best-practices/index.md

```diff
@@ -151,13 +161,25 @@ for buttons! Button elements provide accessible behaviour and built-in form
 submission functionality, and they can easily be styled. There's no point in
 using a `<div>` or some other element pretending to be a button. 
 
-Give the submit button in a sign-up or sign-in form text content that says what it
-does, such as **Create account** or **Sign in**, not **Submit** or **Start**.
+Ensure that the submit button says what it does, such as **Create account** or 
+**Sign in**, not **Submit** or **Start**.
+
+### Ensure successful form submission {: #submission }
+
+Help password managers understand that a form has been submitted. There are two 
+ways to do that: 
+
+* Navigate to a different page.
+* Emulate navigation with `History.pushState()` or `History.replaceState()`, 
+and remove the password form.
+
+If you use `XMLHttpRequest` or fetch, make sure that sign-in success or failure 
```

Comment:
```suggestion
If you use `XMLHttpRequest()` or `fetch()`, make sure that sign-in success or failure 
```

Reply:
`XMLHttpRequest` and `fetch` here are referring to the APIs, not to a constructor or method (so no brackets).

I'll add code-font formatting to fetch for consistency.

See: 
https://web.dev/fetch-upload-streaming
https://web.dev/samesite-cookie-recipes

Reply:
`XMLHttpRequest` and `fetch` here are referring to the APIs, not to a constructor or method (so no brackets).

I'll add code-font formatting to fetch for consistency.

See: 
https://web.dev/fetch-upload-streaming
https://web.dev/samesite-cookie-recipes